### ISSUE TITLE
Uncaught ReferenceError: global is not defined

### ISSUE THREAD

**Issue by rokoroku** [jeffbski/joi-browser#19]
_version_: `joi-browser@10.6.0`

When using bundled `joi-browser.js` or `joi-browser.min.js`, it throws following error:
```
Uncaught ReferenceError: global is not defined
    at Object.t.byteLength (joi.js:1)
    at t (joi.js:1)
    at Object.<anonymous> (joi.js:1)
    at t (joi.js:1)
    at Object.i (joi.js:1)
    at t (joi.js:1)
    at Object.assign.t (joi.js:1)
    at t (joi.js:1)
    at joi.js:1
    at joi.js:1
```
It's for joi-**browser**, but it uses **global** which is an object available only in **Node.js**.

**Comment by jeffbski** [comment]
uggh. thanks for reporting it. We will have to figure out how to work around this.

**Comment by jeffbski** [comment]
This should be fixed in joi-browser@10.6.1
